css/style.css

```diff
@@ -1,19 +1,123 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
-body {
+/* Body */
+body h1{
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+li{
+    list-style: none;
+}
+.logo-svg{
+    height: 40px;
+    width: 40px;
+}
+.flexboxrow{
+    display: flex;
+    
+}
+.navul{
+   
+    justify-content:space-around;
+
+}
+.li-space{
+    width: 50%;
+}
+.nav-li{
+color: gray;
```

Comment:
Indentation is very inconsistent across the styles. Please correct the formatting (VSCode can provide great help with this)